initial/src/main/java/com/example/services/CourseService.java

```diff
@@ -0,0 +1,22 @@
+package com.example.services;
+
+import com.example.springboot.Course;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Date;
+
+@RestController
+public class CourseService {
+    // go through static modifiers & public/private modifiers for java methods
+    public String getCourse(long id) throws JsonProcessingException {
+        if(id > 10 || id < 1) {
+            throw new IllegalArgumentException("The course id does not exist");
```

Comment:
We will be covering the logging system in a later section.  Throwing exception with a customized message is not beneficial to our system at all.

Reply:
got it - sounds like this should be removed entirely for now?